### ISSUE TITLE
The model's property is changed on the UI layer

### ISSUE THREAD

**Issue by hoangnguyen92dn** [miles-pudge-halter/nimble-survey-android#7]
The `coverImageUrl` from `Survey` model is changed on the Fragment, I'd suggest creating a new property for the large image URL instead. What do you think?

https://github.com/miles-pudge-halter/nimble-survey-android/blob/66a7588f2780826d06c4b4b4f315f59e85b495dd/app/src/main/java/me/ayoe/nimble/ui/home/surveyDetails/SurveyDetailsFragment.kt#L34-L35

https://github.com/miles-pudge-halter/nimble-survey-android/blob/66a7588f2780826d06c4b4b4f315f59e85b495dd/app/src/main/java/me/ayoe/nimble/ui/home/survey/SurveyFragment.kt#L44

**Comment by miles-pudge-halter** [comment]
Yes. We should have an attribute like `largeCoverImageUrl` in the UI model and add it during the mapping from data model to UI model.

**Comment by hoangnguyen92dn** [comment]
@miles-pudge-halter Yup. It's a better approach rather than putting the `l` on the UI layer.
